### DISCUSSION TITLE
Is it correct to apply likelihood for making predictions?

### DISCUSSION THREAD

**Discussion by HSID** [cornellius-gp/gpytorch#1699] [Q&A] [answered]
hi folks,
I quickly read through the implementations and found that the DefaultPredictionStrategy has already taken the noise of likelihood into consideration to calculate the posterior distribution. 
However, in the exact GP sample code, the predicted distribution is "likelihood(model(test_x))", which seems like adding the noise term two times to me.

**Comment by Balandat** [comment, marked as answer]
As you said, the prediction strategy will properly take the noise of the likelihood into account when computing the posterior distribution. For prediction, `model(test_x)` provides the posterior over the latent function values. Applying the `likelihood` to that posterior gives you the posterior predictive, i.e. the distribution over the observed values (internally this is done by just by adding the observation noise to the diagonal of the posterior covariance).
